### PR TITLE
Update to the variables

### PR DESCRIPTION
**Pull request by rdzotz**
Looks like some mixup here in the README with the PYTHON_VERSION missing

### CODE REVIEW

**Comment by mariamedp** [comment]
Hi @rdzotz, thanks for spotting this! 
Last week I completed a PR (#23) increasing the Python and SDK versions, but now I see I forgot to change them here and in the [requirements.txt file](https://github.com/microsoft/dstoolkit-mlops-base/blob/main/requirements.txt). Could you please update them in the README to 3.8 and 1.35? And use this PR to update the pinned versions for `azureml-core` and `azureml-opendatasets` in the requirements file as well? Thank you!

File: docs/how-to/GettingStarted.md [hunk 1]
```diff
@@ -9,8 +9,9 @@ To setup your own MLOPs project in you azure subscription, follow these steps:
 - **[AML-related variables](../../configuration/configuration-aml.variables.yml)**: contains the definition of AML-related environment variables
 
 ```
-SDK_VERSION: the version of Azure ML SDK. Default value is 3.7
-AML_DATASET: training dataset name. Default value is 1.27
+PYTHON_VERSION: the version of python. Default value is 3.7
+SDK_VERSION: Default value is 1.27
```
Comment (mariamedp) [inline docs/how-to/GettingStarted.md:13]: Please add here the description: "the version of Azure ML SDK. Default value is..."

**Review by mariamedp — CHANGES_REQUESTED** [review]
Hi @rdzotz, thanks for spotting this! 
Last week I completed a PR (#23) increasing the Python and SDK versions, but now I see I forgot to change them here and in the [requirements.txt file](https://github.com/microsoft/dstoolkit-mlops-base/blob/main/requirements.txt). Could you please update them in the README to 3.8 and 1.35? And use this PR to update the pinned versions for `azureml-core` and `azureml-opendatasets` in the requirements file as well? I also added a small comment. Thank you!

**Comment by rdzotz** [comment]
Hi @mariamedp, I made the requested changes

**Review by mariamedp — APPROVED** [review]

**Comment by mariamedp** [comment]
@rdzotz Thanks! Now you just need to sign the Contributor License Agreement (see bot message above) and we're ready to merge

**Pull request merged**